docs/release_notes/v1.7.0.md

```diff
@@ -0,0 +1,386 @@
+
+> **Note: This release contains a few [breaking changes](#breaking-changes).**
+
+See [this](#upgrading-to-dapr-1.7.0) section on upgrading Dapr to version 1.7.0.
+
+## Acknowledgements
+
+Thanks to everyone who made this release possible!
+
+@1046102779, @AaronCrawfis, @abelperezok, @akkie, @alberthuang24, @amimimor, @amulyavarote, @artursouza, @beiwei30, @berndverst, @callmexiaolu88, @chinzhiweiblank, @chiru1221, @Cowlord711, @CrazyHZM,
+ @cuishuang, @daixiang0, @davidgladstone, @DeepanshuA, @derrickburns, @dmitsh, @duglin, @Gehhilfe, @Giovds, @greenie-msft, @halspang, @hhunter-ms, @hmz777, @ItalyPaleAle, @jigargandhi, @jjcollinge, @Joshswooft, @l0ll098, @LaurenceLiZhixin, @lechnerc77, @li1234yun, @light-pan, @ls-2018, @lucasjellema, @lupino3, @marcinjahn, @marviniter, @msfussell, @mukundansundar, @mumumi, @naveen1100, @newbe36524, @nwiizo, @paulyuk, @pkedy, @pravinpushkar, @psychomelet, @qt-luigi, @r-c-correa, @RichardLaos, @rlfcunha, @rodrigoscna, @SergioVelmay, @shizn, @shivamkm07, @Shoothzj, @shubham1172, @si618, @siredmar, @skyao, @sourikghosh, @surenderssm, @Syuparn, @Taction, @tanvigour, @theonefx, @thielepaul, @uptutu, @uy-andrew, @vfarcic, @willvelida, @wimvdc, @XavierGeerinck, @yaron2, @yauri-io, @yordan-pavlov, @zdianjiang
+
+## New in this release
+
+### Dapr Runtime
+- **FIXED** Dapr setting a default content type of `text/plain` [4472](https://github.com/dapr/dapr/pull/4472)
+- **ADDED** Circuit Breaking [777](https://github.com/dapr/dapr/issues/777)
+- **ADDED** Content-Type for state operations [2892](https://github.com/dapr/dapr/issues/2892)
+- **ADDED** Resiliency policies across all building blocks [3586](https://github.com/dapr/dapr/issues/3586)
+- **CHANGED** Actor Reentrancy to stable feature [3811](https://github.com/dapr/dapr/issues/3811)
+- **CHANGED** gRPC Proxying to stable feature [3814](https://github.com/dapr/dapr/issues/3814)
+- **FIXED** Service invocation to return original status code on service invocation. Add the possibility to configure retries base on error type [3826](https://github.com/dapr/dapr/issues/3826)
+- **FIXED** Serialization of 64bit values. [3837](https://github.com/dapr/dapr/issues/3837)
+- **ADDED** Unsubscribe method for the configuration API. [3906](https://github.com/dapr/dapr/issues/3906)
+- **FIXED** Bug in runtime matching wildcard routes for pubsub. [3920](https://github.com/dapr/dapr/issues/3920)
+- **ADDED** Actor options configuration per actor type [3934](https://github.com/dapr/dapr/issues/3934)
+- **ADDED** Logging for mTLS cert expiration [3940](https://github.com/dapr/dapr/issues/3940)
+- **ADDED** Logging for sidecar pod name in Dapr Operator [3947](https://github.com/dapr/dapr/issues/3947)
+- **FIXED** Method invocation / Dapr automatically decodes %2f in URL [4008](https://github.com/dapr/dapr/issues/4008)
```

Comment:
Should this be a breaking change even thought it is a bug fix ? @artursouza @yaron2